src/main/kotlin/cz/startnet/utils/pgdiff/CLI.kt

```diff
@@ -24,21 +25,33 @@ class CLI : CliktCommand(name = "apgdiff") {
     val oldDumpFile by argument(help = "Path to the original dump file").file(mustBeReadable = true)
     val newDumpFile by argument(help = "Path to the new dump file").file(mustBeReadable = true)
 
+    val schemas: List<String> by option(
+        "-n",
+        "--schema",
+        help = "diff the specified schema(s) only",
+        metavar = "PATTERN"
+    ).multiple()
+
+    val excludeSchemas: List<String> by option(
+        "-N",
+        "--exclude-schema",
+        help = "do NOT diff the specified schema(s)",
+        metavar = "PATTERN"
+    ).multiple()
+
 
     override fun run() {
         val arguments = PgDiffOptions(
             isAddDefaults = addDefaults,
-            dropCascade = dropCascade
+            dropCascade = dropCascade,
+            schemas = schemas,
+            excludeSchemas = excludeSchemas
         )
         val dumpOld = oldDumpFile.bufferedReader(Charset.forName(inCharsetName))
         val dumpNew = newDumpFile.bufferedReader(Charset.forName(inCharsetName))
         val charset = Charset.forName(outCharsetName) ?: error("charset $outCharsetName not found")
 
-        val res = PgDiff.createDiff(
-            dumpOld, dumpNew,
-            outputIgnoredStatements = outputIgnoredStatements,
```

Comment:
doesn't `outputIgnoredStatements` need to be added to `arguments`?

Reply:
good catch!